src/NServiceBus.Core/Licensing/LicenseSources.cs

```diff
@@ -21,30 +21,17 @@ public static LicenseSource[] GetLicenseSources(string licenseText, string licen
                 sources.Add(new LicenseSourceFilePath(licenseFilePath));
             }
 
-            sources.Add(new LicenseSourceConfigFile());
+            sources.Add(new LicenseSourceAppConfigLicenseSetting());
+            sources.Add(new LicenseSourceAppConfigLicensePathSetting());
 
-            sources.Add(new LicenseSourceFilePath(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "NServiceBus", "License.xml")));
-            sources.Add(new LicenseSourceFilePath(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "License", "License.xml")));
```

Comment:
Agree, I'd like to make a clean break here, and have a single convention-based file location for each level: per-project, per-user, machine-global.